v2/protegrity-data-tokenization/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,8 @@
+# Root logger option
+log4j.rootLogger=INFO, stdout
+
+# Direct log messages to stdout
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.sss+Z} [%p]: %m - %c{1}:%L%n
```

Comment:
Could you please add a new line at the end of this file? This is a convention in *nix systems